### PR TITLE
create categories

### PR DESCRIPTION
**Pull request by Maxus94**

### CODE REVIEW

File: src/components/Categories/CategoryList/CategoryList.jsx [hunk 1]
```diff
@@ -1,5 +1,26 @@
+import categoriesList from '../../../temp/categories.json';
+import css from './CategoryList.module.css';
+import sprite from '../../../../src/images/icons.svg'
```
Comment (Iryna-Holova) [inline src/components/Categories/CategoryList/CategoryList.jsx:3]: import icons from '../../../images/icons.svg'

File: src/components/Categories/CategoryList/CategoryList.jsx [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
-  return <ul>CategoryList</ul>;
+  return (
+    <ul className={css.categoriesList}>
+      {categoriesList.map(category => (
+        <li className={css.categoryItem}>
+          <div className={css.categoryLinkContainer}>
+            <a className={css.categoryLink} href="">
+              {category.name}
+            </a>
+            <a className={css.categoryArrow} href="">
```
Comment (Iryna-Holova) [inline src/components/Categories/CategoryList/CategoryList.jsx:14]: <Link to={`recipes/?category=${category.name}`}>

Comment (Iryna-Holova) [inline src/components/Categories/CategoryList/CategoryList.jsx:14]: <Link to={`recipes/?category=${category.name}`}>

Comment (Iryna-Holova) [inline src/components/Categories/CategoryList/CategoryList.jsx:14]: <Link to={`recipes/?category=${category.name}`}>

Comment (Iryna-Holova) [inline src/components/Categories/CategoryList/CategoryList.jsx:14]: Link to={`recipes/?category=${category.name}`}

File: src/components/Categories/CategoryList/CategoryList.jsx [hunk 3]
```diff
@@ -1,5 +1,26 @@
+import categoriesList from '../../../temp/categories.json';
+import css from './CategoryList.module.css';
+import sprite from '../../../../src/images/icons.svg'
+
 const CategoryList = () => {
-  return <ul>CategoryList</ul>;
+  return (
+    <ul className={css.categoriesList}>
+      {categoriesList.map(category => (
+        <li className={css.categoryItem}>
```
Comment (Iryna-Holova) [inline src/components/Categories/CategoryList/CategoryList.jsx:9]: missing key prop

File: src/components/Categories/CategoryList/CategoryList.module.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  flex-direction: column;
+  gap: 16px;
+}
+
+.categoryItem {
+  height: 250px;
+  border-radius: 20px;
+  position: relative;
+  background: rgba(5, 5, 5, 0.18);
+  background-image: url('../../../temp/cat-img.jpg');
```
Comment (Iryna-Holova) [inline src/components/Categories/CategoryList/CategoryList.module.css:12]: should not work like this, use multiple backgrounds instead

File: src/components/Categories/CategoryList/CategoryList.module.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+.categoryLink {
+  height: 40px;
+  padding: 0 12px;
+  border: 1px solid var(--color-white-40);
+  border-radius: 30px;
+  background: var(--color-white-30);
+  display: flex;
+  align-items: center;
+  font-family: var(--font-family);
```
Comment (Iryna-Holova) [inline src/components/Categories/CategoryList/CategoryList.module.css:34]: should be add to css/reset.css. remove please

File: src/components/Categories/CategoryList/CategoryList.module.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+.categoryArrow {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 40px;
+  height: 40px;
+  border: 1px solid var(--color-white-30);
+  border-radius: 30px;
+  stroke: white;
```
Comment (Iryna-Holova) [inline src/components/Categories/CategoryList/CategoryList.module.css:50]: color

**Review by Iryna-Holova — CHANGES_REQUESTED** [review]
fix please

File: src/components/Categories/CategoryList/CategoryList.jsx (hunk 1 above)
Comment (Maxus94) [reply src/components/Categories/CategoryList/CategoryList.jsx:3]: Corrected

File: src/components/Categories/CategoryList/CategoryList.jsx [hunk 7]
```diff
@@ -1,5 +1,26 @@
+import categoriesList from '../../../temp/categories.json';
+import css from './CategoryList.module.css';
+import sprite from '../../../../src/images/icons.svg'
+
 const CategoryList = () => {
-  return <ul>CategoryList</ul>;
+  return (
+    <ul className={css.categoriesList}>
```
[the remark this answers is not in the document]
Comment (Maxus94) [reply src/components/Categories/CategoryList/CategoryList.jsx:7]: Corrected

File: src/components/Categories/CategoryList/CategoryList.jsx (hunk 2 above)
Comment (Maxus94) [reply src/components/Categories/CategoryList/CategoryList.jsx:14]: Corrected

File: src/components/Categories/CategoryList/CategoryList.jsx (hunk 3 above)
Comment (Maxus94) [reply src/components/Categories/CategoryList/CategoryList.jsx:9]: Corrected

File: src/components/Categories/CategoryList/CategoryList.module.css (hunk 4 above)
Comment (Maxus94) [reply src/components/Categories/CategoryList/CategoryList.module.css:12]: Corrected

File: src/components/Categories/CategoryList/CategoryList.module.css (hunk 5 above)
Comment (Maxus94) [reply src/components/Categories/CategoryList/CategoryList.module.css:34]: corrected

File: src/components/Categories/CategoryList/CategoryList.module.css (hunk 6 above)
Comment (Maxus94) [reply src/components/Categories/CategoryList/CategoryList.module.css:50]: Corrected

**Comment by Maxus94** [comment]
Fixed

**Review by Iryna-Holova — APPROVED** [review]
ok

**Review by Iryna-Holova — APPROVED** [review]

**Pull request merged**